types/hapi/index.d.ts

```diff
@@ -42,6 +42,18 @@ import { PolicyOptionVariants, EnginePrototypeOrObject, PolicyOptions, EnginePro
  +                                                                           +
  + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + */
 
+/**
+ * one of
+ * a single plugin name string.
+ * an array of plugin name strings.
+ * an object where each key is a plugin name and each matching value is a
+ * {@link https://www.npmjs.com/package/semver version range string} which must match the registered
+ *  plugin version.
+ */
+export type Dependencies = string | string[] | {
```

Comment:
```suggestion
export type Dependencies = string | string[] | Util.Dictionary<string>;
```

Reply:
Tbh I was never a fan of the `Dictonary` thing, it feels like pointless indirection.
`{ [k: string]: string }`
`Util.Dictionary<string>`

I can change it tho.

Reply:
I don't mind either. I just saw that it was used everywhere else 👍🏻